vendor_skins/Evercast/plugins/obs-outputs/WebRTCStream.cpp

```diff
@@ -286,7 +288,9 @@ bool WebRTCStream::startWebSocket(WebRTCStream::Type type)
 
 		bool successfullyJoined = session_data->awaitJoinComplete(5);
 		if (!successfullyJoined) {
-			recordConnectionError("There was a problem connecting to your Evercast room.  Are attendees present?");
+            if (!this->connection_invalidated) {
+                recordConnectionError("There was a problem connecting to your Evercast room.  Are attendees present?");
```

Comment:
Just to confirm, you're waiting to merge EBS-87 to update this message in this PR, right?

Reply:
Yes, the merge needs to be done manually for this one.